src/Neo.VM/OpCode.cs

```diff
@@ -769,6 +773,8 @@ public enum OpCode : byte
         /// <summary>
         /// The item n back in the stack is moved to the top.
         ///
+        /// <example>a b c d 2 -> a c d b</example>
+        /// index => 3[2]1 0
```

Comment:
Move index inside example? also '>' must be escaped?

Reply:
'>' had been written in \<summary\> everywhere, and it seems escaping is unnecessary. Or, shall I change all the '>'?